packages/date-picker/src/vaadin-infinite-scroller.js

```diff
@@ -69,10 +68,6 @@ class InfiniteScroller extends PolymerElement {
     `;
   }
 
-  static get is() {
-    return 'vaadin-infinite-scroller';
-  }
```

Comment:
Removed custom element definition because `InfiniteScroller` is kind of an abstract class now.